beacon-chain/cache/skip_slot_cache.go

```diff
@@ -14,6 +14,13 @@ import (
 	"go.opencensus.io/trace"
 )
 
+const (
+	// delay parameters for checking in progress requests.
+	minDelay    = float64(10)        // 10 nanoseconds
+	maxDelay    = float64(100000000) // 0.1 second
+	delayFactor = 1.1
+)
+
```

Comment:
These were defined in `attestation_data.go` but shared with skip slot cache. Moving it here now